src/utils/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin set-flex-property {
+  justify-content: space-between;
+  align-items: center;
+}
+
+@mixin buy-btn {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 700;
+  font-size: 14px;
+  height: 40px;
+  line-height: 16px;
+  border-radius: 5px;
+}
+
+@mixin container-property {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+}
```

Comment:
same here, totally not clear what is it about and whether you really need it. Does not seem to be reusable